.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+Thanks for contacting us! Before you submit your issue, please fill out all the
+information below:
+
+- Please provide a detailed description of your problem:
```

Comment:
Nice first cut! I like the "version" reminder below.

For the other 3 questions, there are more than one way to phrase them. This "detailed description" may seems a little too vague and have overlap with the other two.

I remember I saw something like this in code.google.com before.

* What are the steps to reproduce the issue?
  1. 
  2. 
  3. 

* What did you see as a result?

* What would you want to see instead?

The good thing about the above 3 questions is that they can naturally work as a acceptance test for an upcoming bugfix. And they are technically also suitable for new feature request too.

Just my $0.02